src/Storages/StorageFile.cpp

```diff
@@ -125,6 +125,8 @@ namespace ErrorCodes
     extern const int UNSUPPORTED_METHOD;
 }
 
+using String = std::string;
```

Comment:
Why not `#include <Core/Types.h>`?

Reply:
because I'm silly and couldn't find quickly which include contained it :smiling_face_with_tear: 
I'll change it in a future PR to avoid CI rerun